js/views/userPageVw.js

```diff
@@ -543,25 +542,17 @@ UserPageVw = pageVw.extend({
     }
 
     if (state == "listing" || state == "listingOld" || state == "listingNew") {
-      isItemType = true;
-    }
-
-    //set address bar
-    if (isItemType) {
       addressState = "/listing";
+      addressState = hash ? addressState + "/" + hash : addressState;
     } else {
       addressState = "/" + state;
     }
-    currentAddress = this.model.get('page').profile.guid + addressState;
-    currentHandle = currentHandle ? currentHandle + addressState : "";
-    if (isItemType && hash) {
-      currentAddress += "/"+ hash;
-      currentHandle = currentHandle ? currentHandle += "/"+ hash : "";
-    } else if (addressState === "createStore"){
-      currentAddress = this.model.get('page').profile.guid;
-    }
 
-    window.obEventBus.trigger("setAddressBar", {'addressText': currentAddress, 'handle': currentHandle});
+    currentAddress = currentHandle || this.model.get('page').profile.guid;
+    currentAddress += addressState;
+
+    window.obEventBus.trigger("setAddressBar", {'addressText': currentAddress});
+    currentHandle && app.appBar.setTitle(currentHandle);
```

Comment:
That's a good point. It was only there because it uses the same data as the address bar.